examples/creatives/nonlinear_overlay.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * A sample SIMID nonlinear ad that shows an overlay of
```

Comment:
This needs to be commented differently than you might comment production code. Because this is sample code we can be a little more verbose.


An ad built for testing relevant SIMID API for nonlinear SIMID creatives. These API include:
expandCreative - This should expand the creative to take up the entire frame.
collapseCreative - Returns the creative to its original size.
resizeCreative - Allows a specific size for the creative.